apps/fluent-tester/src/TestComponents/Shadow/ShadowDepthTestSection.tsx

```diff
@@ -32,21 +24,12 @@ interface ShadowTestBoxProps {
 
 const ShadowTestBox: React.FunctionComponent<ShadowTestBoxProps> = (props: ShadowTestBoxProps) => {
   const theme = useTheme();
-  const themedStyles = getThemedStyles(theme);
   const backgroundColor = props.isBrand ? theme.colors.brandedBackground : theme.colors.background;
   const textColor = props.isBrand ? theme.colors.primaryButtonText : theme.colors.bodyText;
 
   return (
     <Shadow shadowToken={props.shadowToken}>
-      <View
-        style={[
-          commonTestStyles.view,
-          themedStyles.effectBox,
-          themedStyles.vmargin,
-          themedStyles.padding,
-          { backgroundColor: backgroundColor },
-        ]}
-      >
+      <View style={mergeStyles(styles.shadowTestBox, { backgroundColor: backgroundColor })}>
```

Comment:
Nit: Could we move the style creates / merging to its' own line? I think in general, we should strive to not have inline styles in FluentTester (even If this one is probably fine because `mergeStyles` will memoize)

Reply:
What's the reasoning between not having inline styles? Is it related to the re-render test failing? I.e. if a style object is created inline, re-rendering won't work as expected

Reply:
Tried to fix but still a little unclear with what exactly I'm trying to avoid. Is the issue that mergeStyles is not on its own line, or the issue that we're creating a new backgroundColor object? If the reason why we don't like inline styles is related to re-rendering consistency I'm not sure my changes actually fix the issue

Reply:
Think this has been resolved in some other comments below